elements/bulbs-video/components/revealed.js

```diff
@@ -156,30 +171,96 @@ export default class Revealed extends React.Component {
     return false;
   }
 
-  vastUrl (videoMeta) {
-    let baseUrl = 'http://us-theonion.videoplaza.tv/proxy/distributor/v2?rt=vast_2.0';
-
-    let vastTestId = this.vastTest(window.location.search);
+  getProfValue () {
+    let prof;
+    if (window.isMobile.any) {
+      prof = 'theonion_mobileweb_html5';
+    }
+    else {
+      prof = 'theonion_desktop_html5';
+    }
+    return prof;
+  }
 
-    // AD_TYPE: one of p (preroll), m (midroll), po (postroll), o (overlay)
-    baseUrl += '&tt=p';
-    videoMeta.tags.push('html5'); // Force HTML 5
-    // Tags
-    baseUrl += '&t=' + videoMeta.tags;
-    //Category
-    let hostChannel = videoMeta.hostChannel;
-    let channel = videoMeta.channel_slug;
-    let series = videoMeta.series_slug;
-    let category = `${hostChannel}/${channel}`;
-    if (series) {
-      category += `/${series}`;
+  getDeviceAcronym () {
+    let deviceAcronym;
+    if (window.isMobile.any) {
+      deviceAcronym = 'm';
+    }
+    else {
+      deviceAcronym = 'd';
     }
-    baseUrl += '&s=' + category;
-    baseUrl += '&rnd=' + this.cacheBuster();
+    return deviceAcronym;
+  }
+
+  getSiteName () {
+    let targetChannel = this.props.video.targeting.target_channel;
+    return targetChannel.replace(/-/g, '');
+  }
 
-    if (vastTestId) {
-      baseUrl += '&xgid=' + vastTestId;
+  getDfpSection () {
+    let dfpSection;
+    if (window.TARGETING.dfp_section) {
+      dfpSection = window.TARGETING.dfp_section;
+    }
+    else if (window.TARGETING.dfp_specialcoverage) {
+      let slug = window.TARGETING.dfp_specialcoverage;
+      dfpSection = `specialcoverage_${slug}`;
+    }
+    else {
+      dfpSection = 'video';
     }
+    return dfpSection;
+  }
+
+  buildCustomSiteSectionId (hostChannel) {
+    // format: <device acronym>.<site name>_<dfp section>_<host channel>
```

Comment:
I think we can remove this comment, because that format is clear in the interpolation below